src/Apps/Feature/Components/FeatureSet.tsx

```diff
@@ -47,8 +53,43 @@ export const FeatureSet: React.FC<FeatureSetProps> = ({
         ]}
         gridGap={2}
       >
-        {children}
+        {set.orderedItems.edges.map(({ node: orderedItem }) => {
+          switch (orderedItem.__typename) {
+            case "FeaturedLink":
+              return (
+                <FeatureFeaturedLink
+                  key={orderedItem.id}
+                  featuredLink={orderedItem}
+                />
+              )
+            default:
+              // TODO: Unimplemented: Artist | Artwork | Gene | Sale | PartnerShow | Profile | OrderedSet
+              return null
+          }
+        })}
       </CSSGrid>
     </Container>
   )
 }
+
+export const FeatureSetFragmentContainer = createFragmentContainer(FeatureSet, {
+  set: graphql`
+    fragment FeatureSet_set on OrderedSet {
+      name
+      description
+      itemType
+      # TODO: Handle pagination
+      orderedItems: orderedItemsConnection(first: 50) {
```

Comment:
I'd start with something lower (20?).

Reply:
👍 funnily enough https://github.com/artsy/artsy-backbone-mixins/blob/6309f2a4d4e1a1458105d934b1ec89af6bb87e98/lib/feature.coffee#L113 we should make a call about how to actually handle pagination here though — just infinite scroll? Historically these were never actually paginated.

Reply:
Haha!

Although in the old feature pages these are rendered client-side so not as much of an issue.

I'd say infinite scroll, but there may also be some UX-reason for not supporting long features? We can ask Nicole.

Reply:
One of the unfortunate things about the pagination here is that we need to paginate both sets and the contents inside of them. I do wonder if this is overkill if we never plan on having set content over a specific size and can just set a reasonable limit until a use-case presents itself.

Reply:
Or maybe one of those like truncated see-more gradient fade out buttons would be a good UX for long sets which would help make the infinite scrolling a bit more natural.

Reply:
Interesting interesting...

I'm okay with setting a reasonable default for now.